src/PDMats.jl

```diff
@@ -36,24 +36,24 @@ module PDMats
         Xt_invA_X,
         test_pdmat
 
-    import Base.BLAS: nrm2, axpy!, gemv!, gemm, gemm!, trmv, trmv!, trmm, trmm!
-    import Base.LAPACK: trtrs!
-    import Base.LinAlg: A_ldiv_B!, A_mul_B!, A_mul_Bc!, A_rdiv_B!, A_rdiv_Bc!, Ac_ldiv_B!, Cholesky
-
 
     # The abstract base type
 
     abstract type AbstractPDMat{T<:Real} end
 
+    const HAVE_CHOLMOD = isdefined(SuiteSparse, :CHOLMOD)
+
     # source files
 
-    include("chol.jl")   # make Cholesky compatible with both Julia 0.3 & 0.4
```

Comment:
Come to think of it, if this was only included for compatibility with Julia 0.3 and 0.4, would it be possible to remove it entirely and just use `Cholesky` directly? More of a general wondering though, not something you need to take on here or anything.